src/api.coffee

```diff
@@ -136,4 +136,8 @@ module.exports = (application) ->
 		.catch (err) ->
 			res.status(503).send(err?.message or err or 'Unknown error')
 
+	api.get '/v1/device', (req, res) ->
+		state = device.getState()
+		res.status(200).send(JSON.stringify(state))
```

Comment:
why not `res.json(state)` ?
it sets 200 by default and also the correct headers, and stringifies for you.